electron/src/js/menu/system.ts

```diff
@@ -364,16 +368,18 @@ const processMenu = (template, language) => {
   }
 };
 
-const changeLocale = language => {
+const changeLocale = (language: SupportedLanguage): void => {
   locale.setLocale(language);
   dialog.showMessageBox(
     {
       buttons: [
-        locale[language].restartLater,
-        environment.platform.IS_MAC_OS ? locale[language].menuQuit : locale[language].restartNow,
+        locale.LANGUAGES[language].restartLater || locale.LANGUAGES.en.restartLater,
+        environment.platform.IS_MAC_OS
+          ? locale.LANGUAGES[language].menuQuit || locale.LANGUAGES.en.menuQuit
+          : locale.LANGUAGES[language].restartNow || locale.LANGUAGES.en.restartNow,
```

Comment:
The fallback to English was simpler before. Let's avoid adding it everwhere we use a string